libs/ui/src/big_metric.tsx

```diff
@@ -0,0 +1,57 @@
+/* stylelint-disable order/properties-order */
+import React from 'react';
+import styled from 'styled-components';
+
+import { format } from '@votingworks/utils';
+
+import { H1 } from './typography';
+import { LabelledText } from './labelled_text';
+
+/** Props for {@link BigMetric}. */
+export interface BigMetricProps {
+  label: React.ReactNode;
+  value: number;
+
+  /** For backward compatibility with tests in VxScan */
+  valueElementTestId?: string;
+}
+
+const StyledContainer = styled.span`
+  display: inline-block;
+  font-size: 1rem;
+  line-height: 1;
+  margin-bottom: 0.35rem;
+  text-align: center;
+`;
+
+const StyledValue = styled(H1)`
+  display: inline-block;
+
+  /* TODO: Remove once VxScan is updated to default to the new VVSG size theme: */
+  font-size: ${(p) => (p.theme.sizeMode === 'legacy' ? '72px' : undefined)};
+  line-height: 1;
+  margin: 0;
+`;
+
+/**
+ * Displays a formatted, H1-sized numerical metric value with the given label.
+ */
+export function BigMetric(props: BigMetricProps): JSX.Element {
+  const { label, value, valueElementTestId } = props;
+
+  const formattedValue = format.count(value);
+  const formattedLabel = `${label}: ${formattedValue}`;
+
+  return (
+    <StyledContainer>
+      <LabelledText aria-hidden label={label}>
+        <StyledValue
+          aria-label={formattedLabel}
+          data-testid={valueElementTestId}
```

Comment:
Is there a way to make `<BigMetric data-testid="…"` do what we want? Or is this the better way to do this?

Reply:
Yeah, I believe that would work - I was avoiding having to update the test assertions to include the label text as well, but I'm down to do that folks feel strongly